New package: Graphons v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: Graphons
- Repository: https://github.com/SDS-EPFL/Graphons.jl
- Created by: @dufourc1
- Version: v0.1.0
- Commit: 8498108aee01e67189ae782d9fa8905217c03f9c
- Reviewed by: @dufourc1
- Reference: https://github.com/SDS-EPFL/Graphons.jl/issues/6#issuecomment-3405632082
- Description: Julia package for simple/decorated/probability graphons
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
- Initial release of Graphons.jl
- A package for generating random graphs from simple and decorated (probability) graphons
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598ef7272d7846c443f7dce3a417ab18be0ea677449905f21bbe71ea7b06117fcfddb927bd9b2229f76ffa2fc7f71c5325caa7bb5374edf364c02fcbd548ed890c7d37c9b2a3c461f82778436a52b57d3b5307177bd8889134edf457a412f18ce8818cbec0aae3ac3d5e7ddee15205e9e1179feb090141859df94d1a5b19bbd63da -->

GunnarFarneback commented:
[noblock]

It would be helpful for those checking out the package on GitHub to have some hint of what it's about in the README. Just the Overview section from the documentation would be fine but some of the following sections could also be included.

The name similarity complaint is a false positive. I will override it.

JuliaRegistrator committed:
New package: Graphons v0.1.0

UUID: e0c12bfd-47d7-434e-afb7-632611640ca5
Repo: https://github.com/SDS-EPFL/Graphons.jl.git
Tree: 3562830b0f3c4234dd7f0379b9a49dfc3c8340b0

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
